src/main/java/com/blog/web/mappers/ArticleMapper.java

```diff
@@ -0,0 +1,14 @@
+package com.blog.web.mappers;
+
+import com.blog.web.dto.ArticleDto;
+import com.blog.web.models.Article;
+
+public class ArticleMapper {
```

Comment:
if this is a utility class add a private constructor to preserve singleton nature